CreateChallege page

llJTOll opened the pull request:
챌린지 생성 페이지 껍데기 구현
글자 폰트랑 크기 통일 위해 common/text.css 추가 =>이후 불필요하다면 삭제합시다.
<hr>이 안 먹혀서 이건 후에 수정할게요.
Profile.svelte는 일단은 무시해주세요.

llJTOll committed:
CreateChallege page

챌린지 생성 페이지 껍데기 구현
<hr>이 안 먹혀서 이건 후에 수정할게요.
Profile.svelte는 일단은 무시해주세요.

MOBUMIN commented on src/common/text.css:
```diff
@@ -0,0 +1,30 @@
+.--title-text{
```
혹시 css 변수 명 규칙은 어떤거 쓰신건가욥? 처음 보는 거라서요!

llJTOll replied:
colorVariable.css보고 따라쓴다고 쓴건데 지금보니까 잘못 사용한거네요; 일반 변수명처럼 수정하겠습니다.

MOBUMIN commented on src/common/text.css:
```diff
@@ -0,0 +1,30 @@
+.--title-text{
+    font-family: 'Noto Sans KR';
```
폰트패밀리 이걸로 통일할거면 전체 적용되게 가능할까요? (body태그에 걸고 루트에서 걸어준다던지..)

llJTOll replied:
일단 Home에서 쓰인 폰트로 넣은건데, 따로따로 바꾸기 쉽게 일단 만들어둔거라서 폰트 상의해서 결정하고 바꾸는게 좋을 것 같습니다.

MOBUMIN reviewed (commented):
혹시 버튼 컴포넌트가 빡치게 하면 그냥 버튼 컴포넌트 내용 수정해주셔도 돼요!
완전 굿굿 답글 달리는것만 보고 바로 어푸르브 날리겠습니다

tnghd5761 commented:
1. 'Cancel' 인데 'Cancle' 로 오타가 났네요

2. content가 좌측 상단으로 너무 치우쳐져 있는 감이 있네요. padding을 통해서 가운데로 맞추기 보다는 내용들을 적당한 width의 div 하나로 감싸서 그 div를 중앙 정렬 시키는 것이 좋을 듯합니다. 상단에는 약간의 padding 혹은 margin를 추가해 여유를 좀 주고요. 그리고 지금도 그렇지만 앞으로도 페이지를 구성할 때 'components'폴더 내에 있는 'Container.svelte'(고정된 width로 감싸주기 위한 만든 컴포넌트)를 사용해서 내용을 한 번 감싸주시면 감사하겠습니다!

3. public / private을 선택하는 체크박스를 세로 가운데 정렬해서 옆에 내용과 높이를 좀 맞추면 좋을 것 같습니다!

llJTOll commented:
1. 수정했습니다.
2. 중앙정렬 시키고 padding 넣어서 여유 넣었습니다! Container 미적용한 상태로 정렬시켰는데 혹시 사용해야하는 이유를 자세히 알 수 있을까요? 고정된 가로길이를 가져야 하는 이유를 잘 모르겠어서요...
3. 수정하려고 찾아보는데 쌩으로 padding 줘서 맞추는 방법말고는 아직 못찾아서...  방법 찾는대로 수정하겠습니다.

llJTOll committed:
Create Challenge 수정1

1. test.css 변수명 변경했습니다.
2. text input components 추가했습니다.
3. hr 가로줄 안보이던거 수정했습니다.
4. 가로 중앙정렬 했습니다.

tnghd5761 commented on src/pages/CreateChallenge.svelte:
```diff
@@ -0,0 +1,108 @@
+	}
+
+	.Content{
+		weight: auto;
```
이것도 오타인가요?

tnghd5761 commented on src/pages/CreateChallenge.svelte:
```diff
@@ -0,0 +1,108 @@
+		margin-top:0.1rem;
+	}
+
+	.Contain{
+		height: 2.5rem;
+		display: flex;
+		flex-direction: row;
+		margin-bottom: 0.5rem;
+	}
```
```suggestion
	.Contain{
		height: 2.5rem;
		display: flex;
		flex-direction: row;
		margin-bottom: 0.5rem;
		align-items: center;
	}
```
3.
align-items: center;
이거 한 줄만 추가해주시면 됩니다!
+ global.css 가 input 등에 maring-bottom 이 약간 부여되어 있어서 완전하게 세로 가운데 정렬은 안되네요. global.css를 추후 일부수정할 필요가 있어보입니다!

tnghd5761 reviewed (commented):
2.
+ 고정너비를 가져야 하는 이유는 페이지들의 content 너비를 통일시켜서 사이트에 시각적 안정감과 통일성을 주기 위함입니다. 
+ Container를 사용을 권장하는 이유는 각각 페이지에서 고정된 너비를 매번 부여하는 번거로움을 덜어줌과 동시에 코드의 반복을 줄이고 재사용성을 높이기 위함입니다.

llJTOll committed:
Merge branch 'develop' into CreateChallenge
